frontend/src/screens/studyBanner.tsx

```diff
@@ -0,0 +1,191 @@
+        0
+    );
+
+    const totalPointsEarned = studies.reduce(
+        (sum: number, study: ParticipantStudy) =>
+            sum + (study.learningPath?.completed ? (study.totalPoints || 0) : 0),
+        0
+    );
+
+    const hasCompletedStudies = filteredStudies.some((study) => study.completedAt);
+
+    const fiveMinuteStudies = filteredStudies.filter(
+        (study) => study.stages && study.stages[0]?.durationMinutes === 5
+    );
+
+    const startRandomFiveMinuteStudy = useCallback(() => {
+        if (fiveMinuteStudies.length > 0) {
+            const randomIndex = Math.floor(Math.random() * fiveMinuteStudies.length);
+            const randomStudy = fiveMinuteStudies[randomIndex];
+            navigate(`/studies/details/${randomStudy.id}`);
+        }
+    }, [fiveMinuteStudies, navigate]);
+
+    const formatValue = (value: number) => (value < 10 ? `0${value}` : value);
+
+    const containerStyle: React.CSSProperties = {
+        display: 'flex',
+        justifyContent: 'space-between',
+        alignItems: 'center',
+        padding: '30px',
+        borderRadius: '8px',
+        margin: '0 auto',
+        maxWidth: '1200px',
+        [`@media (max-width: ${theme.breakpoints.sm}px)`]: {
+            flexDirection: 'column',
+            padding: '20px',
+            margin: '0 1%',
+        },
+    };
+
+    const achievementStyle = (isLeft: boolean): React.CSSProperties => ({
+        flex: 1,
+        marginRight: isLeft ? '134px' : '0',
+        marginLeft: '0',
+        [`@media (max-width: ${theme.breakpoints.sm}px)`]: {
+            flex: '1 1 100%',
+            margin: '10px 0',
+        },
+    });
+
+    const titleStyle: React.CSSProperties = {
+        fontWeight: 'bold',
+        fontSize: '24px',
+        marginBottom: '10px',
+        [`@media (max-width: ${theme.breakpoints.sm}px)`]: {
+            fontSize: '1.3em',
+        },
+    };
+
+    const subtitleStyle: React.CSSProperties = {
+        fontWeight: 'bold',
+        fontSize: '16px',
+        marginBottom: '5px',
+        textAlign: hasCompletedStudies ? 'center' : 'left',
+    };
+
+    const textStyle: React.CSSProperties = {
+        marginBottom: '10px',
+        fontSize: '14px',
+        whiteSpace: 'pre-line',
+    };
+
+    const linkStyle: React.CSSProperties = {
+        color: colors.blue,
+        textDecoration: 'none',
+        fontWeight: 'bold',
+        cursor: 'pointer',
+        fontSize: '14px',
+    };
+
+    const numberContainerStyle: React.CSSProperties = {
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+        height: '80px',
+        flex: 1,
+        marginBottom: '10px',
+    };
+
+    const numberTextStyle: React.CSSProperties = {
+        color: colors.purple,
+        fontFamily: 'Helvetica Neue, sans-serif',
+        fontWeight: 200,
+        fontSize: '70px',
+        textAlign: 'center',
+    };
+
+    return (
+        <Group align="flex-start" style={containerStyle}>
```

Comment:
in general we'd like to avoid having these inline styles (defined above in the objects) and just rely on the mantine components to take care of the styling (as much as possible)